srcpkgs/fteqw/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'fteqw'
+pkgname=fteqw
+version=5808
+revision=1
+hostmakedepends="subversion pkgconf"
+makedepends="zlib-devel gnutls-devel libpng-devel libglvnd-devel libXrandr-devel libXcursor-devel alsa-lib-devel libjpeg-turbo-devel opus-devel speex-devel libvorbis-devel freetype-devel"
+short_desc="FTE Quake engine"
+maintainer="Gibson <ypsvlq@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://fte.triptohell.info"
+
+do_fetch() {
+	svn co -r$version svn://svn.code.sf.net/p/fteqw/code/trunk fteqw-$version
+}
```

Comment:
I'm strongly against adding yet another package that pulls from repositories. Don't they at least have tarballs from source revision? Though inactive upstream that doesn't make releases already fills me with dread a bit.

Reply:
It's developed actively but doesn't provide source tarballs except for the actual latest revision which obviously is against policy. If it's an issue then I guess this should be closed for now until upstream provides releases that work.